script.js

```diff
@@ -0,0 +1,10 @@
+let userName = prompt("hello! what is your name?");
+let userAge = prompt(`Hi ${userName}! how old are you?`)
+userAge = Number(userAge);
```

Comment:
Put empty lines between code blocks to make it easier to read
```suggestion
userAge = Number(userAge);

```